docs/Plasma-Mobile-on-Unleashed.md

```diff
@@ -37,15 +37,19 @@ Follow the usual steps you use to build Yocto images. There are more details on
 Once you know you can build for the ```freedom-u540``` machine you should add these lines to your local.conf or distro:
 
 ```
-DISTRO_FEATURES_freedom-u540          += " wayland x11 opengl xattr gles directfb"
-DISTRO_FEATURES_freedom-u540          += " polkit kde pam pulseaudio"
-FEATURE_PACKAGES_append_freedom-u540   = " wayland mesa-megadriver"
-FEATURE_PACKAGES_append_freedom-u540   = " mesa-driver-swrast"
-IMAGE_INSTALL_append_freedom-u540      = " wayland pulseaudio pulseaudio-server"
+## Required DISTRO_FEATURES
+DISTRO_FEATURES_freedom-u540   += " wayland x11 kde opengl pam polkit xattr"
```

Comment:
this seems not correct, it should not be using MACHINE override

Reply:
As in remove the _freedom-u540?

Reply:
yes